### ISSUE TITLE
V10

### ISSUE THREAD

**Issue by docphil30** [rwanoux/channel-fear#2]
Any possibily to have this wonderful adaptation in V10

**Comment by rwanoux** [comment]
Bonjour, (oui j'utilise le français je me dis que tu dois être français..?..)
Ce system est à l'abandon mais DJleChuck en a refait très chouette en repartant de zéro ; tu le trouvera ici : https://foundryvtt.com/packages/channel-fear

**Comment by docphil30** [comment]
Oui, je sais mais j'aimais bien le tien qui est plus complet (il me semble) et surtout j'ai refait les épisodes sous ton système... en tout cas merci pour le travail effectué!
